Sources/AWSLambdaRuntimeCore/LambdaRunner.swift

```diff
@@ -95,7 +95,7 @@ private extension Lambda.Context {
 }
 
 // TODO: move to nio?
-private extension EventLoopFuture {
+extension EventLoopFuture {
```

Comment:
is this change leftover from previous iteration?

Reply:
no it's needed, since I use `mapResult` in another file.

Reply:
Also: Please don't use any access qualifiers (especially not `public`) in front of `extension`. It makes code reviews really difficult because a `func foo() {}` can be public if maybe 100 lines above there's a `public extension XYZ`. I think all qualifiers should be banned in front of `extension`

Reply:
I hope there's some linter for this perhaps? Maybe time to look around for one hm